src/cmd/package.go

```diff
@@ -102,6 +103,28 @@ var packageDeployCmd = &cobra.Command{
 	},
 }
 
+var packageMirrorCmd = &cobra.Command{
+	Use:     "mirror-resources [ PACKAGE ]",
+	Aliases: []string{"mr"},
+	Short:   lang.CmdPackageMirrorShort,
+	Long:    lang.CmdPackageMirrorLong,
+	Args:    cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		pkgConfig.PkgOpts.PackagePath = choosePackage(args)
+
+		pkgConfig.PkgSource = pkgConfig.PkgOpts.PackagePath
+
+		// Configure the packager
+		pkgClient := packager.NewOrDie(&pkgConfig)
+		defer pkgClient.ClearTempPaths()
+
+		// Deploy the package
+		if err := pkgClient.Mirror(noImgChecksum); err != nil {
+			message.Fatalf(err, lang.CmdPackageDeployErr, err.Error())
+		}
```

Comment:
For consistency `noImgChecksum` should be made into `MirrorOpts`, as all other top level packager receivers do not take arguments and instead grab from the opts struct.

Reply:
Yeah that was from back when that wasn't the case - will refactor